eox_core/api/v1/tests/test_users.py

```diff
@@ -4,13 +4,94 @@
 from django.contrib.auth.models import User
 from django.test import TestCase, override_settings
 from django.urls import reverse
-from mock import patch
+from mock import MagicMock, patch
 from rest_framework import status
+from rest_framework.serializers import ValidationError
 from rest_framework.test import APIClient
 
-from eox_core.api.v1.serializers import EdxappUserQuerySerializer
+from eox_core.api.v1.serializers import EdxappUserQuerySerializer, WrittableEdxappUserSerializer
 
+EXTENDED_PROFILE_FIELDS = [
+    "father_last_name",
+    "mother_last_name",
+    "type_document",
+    "personal_id",
+    "mobile",
+    "address",
+]
 
+EDNX_CUSTOM_REGISTRATION_FIELDS = [
+    {
+        "label": "Father last name",
+        "name": "father_last_name",
+        "type": "text",
+    },
+    {
+        "label": "Mother last name",
+        "name": "mother_last_name",
+        "type": "text",
+    },
+    {
+        "label": "Document type",
+        "name": "type_document",
+        "options": [
+            "dni",
+            "passport",
+            "license",
+        ],
+        "type": "select",
+        "default": "dni",
+    },
+    {
+        "label": "Document number",
+        "name": "personal_id",
+        "type": "text",
+    },
+    {
+        "label": "Mobile",
+        "name": "mobile",
+        "type": "text",
+        "restrictions": {
+            "max_length": 12,
+        }
+    },
+    {
+        "label": "Address",
+        "name": "address",
+        "type": "text",
+    },
+]
+
+REGISTRATION_EXTRA_FIELDS = {
+    "address": "required",
+    "country": "required",
+    "father_last_name": "required",
+    "goals": "required",
+    "mailing_address": "hidden",
+    "mobile": "required",
+    "mother_last_name": "required",
+    "personal_id": "required",
+    "type_document": "required",
+}
+
+
+@override_settings(extended_profile_fields=EXTENDED_PROFILE_FIELDS)
```

Comment:
Im pretty sure you can user `@override_settings` once and pass al the settings as params

Reply:
Done!